apiaudio/helper_classes.py

```diff
@@ -74,3 +76,36 @@ def download(
             local_filenames.append(local_filename)
 
         return local_filenames
+
+
+class UploadableResource(APIRequest):
+    @classmethod
+    def upload(cls, file_path: str = "", tags: str = ""):
+        payload = open(file_path, "rb")
```

Comment:
try catch this to show a proper error if file doesnt exist or there are permission issues

Reply:
we can leave .open() default exceptions in my opinion